src/Web/Grand.Web/Features/Handlers/Catalog/GetSearchAutoCompleteHandler.cs

```diff
@@ -126,8 +126,12 @@ public async Task<IList<SearchAutoCompleteModel>> Handle(GetSearchAutoComplete r
                     Product = item,
                     Store = request.Store
                 }, cancellationToken);
-
-                var price = displayPrices ? await PreparePrice(item, request) : (Price: string.Empty, PriceWithDiscount: string.Empty);
+				var displayPricesItem = displayPrices;
```

Comment:
Could you update code:
var displayPricesItem = !item.CallForPrice && displayPrices;

Reply:
Yes, good idea, I've just done it.